force-app/main/default/classes/DonationHistoryController.cls

```diff
@@ -67,18 +71,37 @@ public with sharing class DonationHistoryController {
         DonationHistoryService service = new DonationHistoryService(contactId);
         try {
             List<DonationHistoryView> donationHistories = new List<DonationHistoryView>();
-            donationHistories.addAll(buildHistoryFromOCRs(service.getDonationHistory()));
+            donationHistories.addAll(buildHistoryFromOPPs(service.getDonationHistory(), contactId));
             return donationHistories;
         } catch(AuraHandledException ex) {
             throw new AuraHandledException(ex.getMessage());
         }
     }
+    
+    /** 
+    * @author Salesforce.org
+    * @date 2021
+    * @description method to return whether or not Payments are enabled in org
+    * @return boolean isPaymentEnabled
+    */
+    @AuraEnabled(cacheable=true)
+    public static boolean arePaymentsEnabled(Id contactId){
+        DonationHistoryService service = new DonationHistoryService(contactId);
+		return service.checkIfPaymentsAreEnabled();
+    }
 
-    private static List<DonationHistoryView> buildHistoryFromOCRs(List<Opportunity> contactRoles) {
+    /** 
+    * @author Salesforce.org
+    * @date 2021
+    * @description method creates list of DonationHistoryView to return to LWC
+    * @return List<DonationHistoryView>
+    */
+    private static List<DonationHistoryView> buildHistoryFromOPPs( List<Opportunity> opportunities, Id contactId) {
+        boolean isPaymentEnabled = arePaymentsEnabled(contactId);
         List<DonationHistoryView> donationHistories = new List<DonationHistoryView>();
-        for(Opportunity role : contactRoles) {
-            donationHistories.add(new DonationHistoryView(role));
-        }
+        for (Opportunity opp : opportunities) {
+			donationHistories.add(new DonationHistoryView(opp, (isPaymentEnabled ? opp.npe01__OppPayment__r.get(0).npe01__Payment_Method__c : EMPTY_PAYMENT_METHOD)));
+		}
```

Comment:
be sure the closing bracket is in the same level of the for

Reply:
I think you are indenting with tabs. Configure your IDE for replacing tabs for 4 spaces. Tabs are not interpreted concistenly thru platforms and tools(like github)